src/components/application_manager/include/application_manager/message_helper.h

```diff
@@ -259,6 +259,23 @@ class MessageHelper {
   static smart_objects::SmartObjectList CreateAddCommandRequestToHMI(
       ApplicationConstSharedPtr app, ApplicationManager& app_mngr);
 
+  static smart_objects::SmartObjectSPtr CreateMessageForHMI(
```

Comment:
@IGapchuk please provide description for this function

Reply:
@AKalinich-Luxoft Added in 11b04298dca9cf017301e7cf54e3b206cd43d883.